docs/COMPATIBILITY_MATRIX.md

```diff
@@ -0,0 +1,22 @@
+## Compatibility
```

Comment:
This should be an `h1` header (a single `#`)

Reply:
done!